examples/README.md

```diff
@@ -0,0 +1,67 @@
+# Example connectors
+
+The Native SDK for Connectors provides a couple of examples that show a general pattern, recommended by Snowflake, of 
+creating new connectors as a [Native Applications][Native Apps docs]. 
+From these examples, you will learn:
+* how to create pull-based connectors ([learn more about pull-based pattern][Pull-based pattern])
+* how to create push-based connectors ([learn more about push-based pattern][Push-based pattern])
+* how to implement the connector configuration mechanism
+* how to implement the ingestion mechanism
+* how to create an instance of the application in the Snowflake environment
+* how to use the [SnowSQL][SnowSQL guide] tool in connector development process
+
+## Prerequisites
+
+- Basic knowledge of [Snowflake Native Apps][Native Apps docs]
+- Basic knowledge of SQL/SnowflakeScripting
```

Comment:
here https://docs.snowflake.com/en/developer-guide/snowflake-scripting/index snowflake scripting is written separately, maybe also we can add url to the Snowflake scripting docs?